backend/src/Edna/ExperimentReader/Parser.hs

```diff
@@ -13,7 +13,7 @@ import qualified Data.Map as M
 import Codec.Xlsx (CellValue(..), Worksheet(..), cellValue, ixCell, toXlsxEither, wsCells, xlSheets)
 import Data.List.Split (wordsBy)
 import Data.Text (replace)
-import Lens.Micro (ix, _Just)
+import Lens.Micro (_Just, ix)
```

Comment:
If you run `stylish-haskell -i src/Edna/ExperimentReader/Parser.hs` from the `backend/` folder does it change this line? If yes, what's your `stylish-haskell` version?

Reply:
Yes, it does. `0.11.0.0`

Reply:
Mine is `0.12.2.0`, probably they changed something :shrug: 0.12 was a major update because they switched from `haskell-src-exts` to `ghc-lib-parser`. So I think some subtle changes are expected. If it only affects corner cases like this one (when an identifier starts with `_`), I think it's minor and we don't need to care.